src/rpc_client/state_ops.rs

```diff
@@ -99,6 +99,13 @@ impl ApiInfo {
         RpcRequest::new(STATE_MINER_PROVING_DEADLINE, (miner, tsk))
     }
 
+    pub fn state_miner_available_balance_req(
+        miner: Address,
+        tsk: ApiTipsetKey,
+    ) -> RpcRequest<TokenAmount> {
+        RpcRequest::new_v1(STATE_MINER_AVAILABLE_BALANCE, (miner, tsk))
```

Comment:
`StateMinerAvailableBalance` is a v0 method. (When methods are available in both V0 and V1, we should use V0.)

```suggestion
        RpcRequest::new(STATE_MINER_AVAILABLE_BALANCE, (miner, tsk))
```